charts/ccd-data-store-api/values.yaml

```diff
@@ -88,4 +88,4 @@ java:
 
     DATA_STORE_CROSS_JURISDICTIONAL_ROLES: caseworker-caa,caseworker-approver,next-hearing-date-admin
     DATA_STORE_CITIZEN_ROLES: citizen,letter-holder
-    DATA_STORE_SECURITY_LOGGING_FILTER_PATH_REGEX: '.*'
+    DATA_STORE_SECURITY_LOGGING_FILTER_PATH_REGEX: 'DISABLED'
```

Comment:
The same information that this filter logs is also logged elsewhere - adds no extra benefit.
Could add a proper regex that never matches anything like `$^` but suggesting just the word `DISABLED` for readability :)